clusterloader2/pkg/measurement/common/slos/api_responsiveness_prometheus.go

```diff
@@ -67,6 +67,12 @@ type apiResponsivenessMeasurementPrometheus struct {
 func (a *apiResponsivenessMeasurementPrometheus) Execute(config *measurement.MeasurementConfig) ([]measurement.Summary, error) {
 	var summaries []measurement.Summary
 
+	if config.PrometheusFramework == nil {
+		klog.Errorf("%s: prometheus framework is not provided!")
+		// TODO(krzysied): for the testing purpose metric is not returning error.
```

Comment:
Instead of TODO(krzysied), please open an issue and do TODO(#issue)

Reply:
Done.